health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -78,34 +77,46 @@ export const UICustomizations = {
       return data;
     },
     additionalCustomizations: (row, key, column, value, t, searchResult) => {
+
       switch (key) {
         case "ACTIONS":
+          // TODO : Replace dummy file id with real file id when API is ready
+          const dummyFile = "c22a7676-d5d7-49b6-bcdb-83e9519f58df"
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)